lib/client.js

```diff
@@ -105,6 +104,9 @@ class Client {
     });
   }
 
+  // Convert cookie string from request`s headers to client.cookies object,
```

Comment:
Backtick is not an apostrophe:
```suggestion
  // Convert cookie string from request's headers to client.cookies object,
```